common/lib/xmodule/xmodule/video_module/transcripts_utils.py

```diff
@@ -579,6 +579,8 @@ def get_video_transcript_content(edx_video_id, language_code):
     edx_video_id = clean_video_id(edx_video_id)
     if edxval_api and edx_video_id:
         transcript = edxval_api.get_video_transcript_data(edx_video_id, language_code)
+        if transcript and 'content' in transcript:
+            transcript['content'] = transcript['content'].decode('utf-8')
```

Comment:
Since `edxval_api` is an internal abstraction layer between LMS/CMS and video data, I would imagine that we would have the decoding happen within VAL - and let the VAL interface send/receive unicode.  So in this case, `transcript['content']` would have already been decoded.

Reply:
That is what I attempted first as well but the VAL layers were more complex to fix and were causing further failures over there.  VAL in general is working with byte strings for various other reasons and with files often so there would be a lot more to fix over there.  I opted for this as the cleaner solution for now though agree that that would be the more correct one long term.

Reply:
I see.  In that case, we can have an invariant that VAL will return bytestrings instead of unicode - so callers can expect that consistency.